### PR TITLE
Add a pin for DBarbarian

### LINKED ISSUE #7277: Add me please
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by DBarbarian**
This is a change to add a pin for DBarbarian as instructed by @githubteacher closes #7277

### CODE REVIEW

**Comment by stefanieisidora** [comment]
@DBarbarian Looks perfect!

**Comment by bas** [comment]
@sudipju kind bump, but can you please have your 👀  on this?

File: _pins/DBarbarian.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: DBarbarian
```
Comment (bas) [inline _pins/DBarbarian.json]: Is this your handle?

File: _pins/DBarbarian.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+latitude: 59.3247
+longitude: 18.056
+---
```
Comment (bas) [inline _pins/DBarbarian.json]: @DBarbarian great that you thought about these dashes 👍

File: _pins/DBarbarian.json (hunk 1 above)
Comment (DBarbarian) [inline _pins/DBarbarian.json]: But of course!

File: _pins/DBarbarian.json (hunk 2 above)
Comment (DBarbarian) [inline _pins/DBarbarian.json]: Isn't it

File: _pins/DBarbarian.json (hunk 1 above)
Comment (DBarbarian) [inline _pins/DBarbarian.json]: @bas

Comment (bas) [inline _pins/DBarbarian.json]: Great, I think it is great this way! 🚀

**Pull request merged**